crypto/kzg4844/kzg4844_gokzg.go

```diff
@@ -49,7 +49,7 @@ func gokzgInit() {
 func gokzgBlobToCommitment(blob Blob) (Commitment, error) {
 	gokzgIniter.Do(gokzgInit)
 
-	commitment, err := context.BlobToKZGCommitment((gokzg4844.Blob)(blob), 0)
+	commitment, err := context.BlobToKZGCommitment((*gokzg4844.Blob)(&blob), 0)
```

Comment:
```suggestion
	commitment, err := context.BlobToKZGCommitment((*gokzg4844.Blob)(blob), 0)
```

... and change argument to `blob *Blob`.

It's better to be consistent with upstream, that way it will be easier to sync changes later on. See:

https://github.com/ethereum/go-ethereum/blob/acd1eaae2c5006dd7f5ae42455bc7f61e5471013/crypto/kzg4844/kzg4844_gokzg.go#L52